components/ui/BannerCarousel.tsx

```diff
@@ -207,11 +222,27 @@ function BannerCarousel(props: Props) {
       class="grid grid-cols-[48px_1fr_48px] sm:grid-cols-[120px_1fr_120px] grid-rows-[1fr_48px_1fr_64px]"
     >
       <Slider class="carousel carousel-center w-full col-span-full row-span-full gap-6">
-        {images?.map((image, index) => (
-          <Slider.Item index={index} class="carousel-item w-full">
-            <BannerItem image={image} lcp={index === 0 && preload} />
-          </Slider.Item>
-        ))}
+        {images?.map((image, index) => {
```

Comment:
try rebasing. This was already improved in master

Reply:
The work was actually being done in parallel, so I have doubts as to whether what is here would be interesting to maintain or what comes from the base